Moya.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveCocoa" do |ss|
     ss.source_files = "Moya/ReactiveCocoa/*.swift"
     ss.dependency "Moya/ReactiveCore"
-    ss.dependency "ReactiveCocoa", "~> 4.0-alpha.1"
+    ss.dependency "ReactiveCocoa", "= 4.0.0-alpha-3"
```

Comment:
Ah, I see. Well, I'd still like to avoid binding to a specific version. Can we still use `"~> 4.0.0-alpha.3"` here, to accommodate for a 4th alpha?

Then again, if the 4th alpha itself has breaking changes... It might make sense to keep this as you have it. Suggestions, @orta?